registry-manager-app/src/main/resources/application.yml

```diff
@@ -33,3 +33,8 @@ spring:
         jdbc:
           lob:
             non_contextual_creation: true
+
+logging.level.de.adorsys.registry.manager.client.AspspOutboundClient: DEBUG
+
+xs2a-adapter:
+  url: http://localhost:8999/v1/aspsps/csv
```

Comment:
for this property, I propose to leave it like that 
```
xs2a-adapter:
  url: http://localhost:8999
```
and rest part of uri move to the client interface

Reply:
done